packages/tradershub/src/providers/SignupWizardProvider/SignupWizardContext.tsx

```diff
@@ -44,6 +44,9 @@ export const SignupWizardContext = createContext<TSignupWizardContext>({
     state: {
         currency: '',
     },
+    reset: /* noop */ () => {
+        /* noop */
+    },
```

Comment:
[Question]: What is the `noop` for?

Reply:
no-operations: used to define empty functions as initial values for the context. A keyword used in typescript as our ts config doesn't allow empty functions to be defined